tests/unit/workflows/test_workflows_tasks_actions.py

```diff
@@ -426,8 +427,12 @@ def test_prepare_update_payload_overwrites():
 
 @patch('inspirehep.modules.workflows.tasks.actions.get_pdf_in_workflow')
 def test_refextract_from_pdf(mock_get_pdf_in_workflow):
-    mock_get_pdf_in_workflow.return_value = pkg_resources.resource_filename(
-        __name__, os.path.join('fixtures', '1704.00452.pdf'))
+    mock_get_pdf_in_workflow.return_value = get_uri(
+        pkg_resources.resource_filename(
+            __name__,
+            os.path.join('fixtures', '1704.00452.pdf'),
+        )
+    )
```

Comment:
It sounds like this wasn't correct before... why would `get_pdf_in_workflow` return a filename? Perhaps it should have been `pkg_resources.resource_file` here, and then we no longer need the `get_uri` ?

Reply:
`get_pdf_in_workflow` retrieves the filename of the pdf that is in the workflow. `get_uri` is required there because the `refextract` task expects that file to be a temporary one, and removes it when done.